Directory.Build.props

```diff
@@ -88,7 +88,7 @@
     <MicrosoftGraphBetaVersion>4.57.0-preview</MicrosoftGraphBetaVersion>
     <MicrosoftExtensionsHttpVersion>3.1.3</MicrosoftExtensionsHttpVersion>
     <MicrosoftIdentityAbstractionsVersion>7.0.0</MicrosoftIdentityAbstractionsVersion>
-    <NetNineRuntimeVersion>9.0.0-preview.7.24405.7</NetNineRuntimeVersion>
+    <NetNineRuntimeVersion>9.0.100-rc.1.24452.12</NetNineRuntimeVersion>
```

Comment:
the line below needs to be updated too but a difference RC version. line 92.

Reply:
Thank you, bumped it to [9.0.0-rc.1.24452.1](https://dotnet.microsoft.com/en-us/download/dotnet/9.0#:~:text=NET%20Core%20Runtime-,9.0.0%2Drc.1.24452.1,-.NET%20Desktop%20Runtime)

Also found another spot that needed to be bumped